changelogs/unreleased/bump-openssl-to-111n.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Bump OpenSSL from 1.1.1f to 1.1.1n for static build (gh-6947).
```

Comment:
In the future, please write release notes in Past Simple, see https://www.tarantool.io/en/doc/latest/contributing/release_notes/

Merging as is, because half of our release notes use Present Simple so we need to revise them anyway.

Reply:
Ouch, it was unintentional. I'm co-author of the document you linked and I usually use this style.